tests/models/test_models_vae.py

```diff
@@ -285,6 +285,18 @@ def get_sd_vae_model(self, model_id="hf-internal-testing/taesd-diffusers", fp16=
         model.to(torch_device).eval()
         return model
 
+    def test_tae_tiling(self):
+        model = self.get_sd_vae_model()
+        model.enable_tiling()
+        in_shapes = [(1, 4, 73, 97), (1, 4, 97, 73), (1, 4, 49, 65), (1, 4, 65, 49), (1, 4, 49, 49)]
+        out_shapes = [torch.Size([1, 3, 584, 776]), torch.Size([1, 3, 776, 584]), torch.Size([1, 3, 392, 520]), 
+                      torch.Size([1, 3, 520, 392]), torch.Size([1, 3, 392, 392])]
```

Comment:
I think we could make use of `parameterized` here to make it a bit cleaner? 

Here's an example: 
https://github.com/huggingface/diffusers/blob/a7de96505b8c71a626c6fa45ff3cf85e42db4960/tests/models/test_models_vae.py#L338C1-L345C6